### PR TITLE
Implemented fragments in Dictionary Activity

### PR DESCRIPTION
**Pull request by chanmol1999**
Implemented fragments along with animation for search button.

### CODE REVIEW

File: app/src/main/java/com/dsciitp/shabd/Dictionary/DictionaryActivity.java [hunk 1]
```diff
@@ -88,73 +92,73 @@ public void onInit(int status) {
...
-                def.append(js.getString("word"));
-                //def.append("/n" + js.getJSONArray("meaning"));
-                meaning.setText(def);
-
-            } catch (JSONException e) {
-                e.printStackTrace();
-            }
-        }
     }
+
```
Comment (atm1504) [inline app/src/main/java/com/dsciitp/shabd/Dictionary/DictionaryActivity.java]: Remove extra spaces

**Review by atm1504 — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/dsciitp/shabd/Dictionary/DictionaryActivity.java [hunk 2]
```diff
@@ -1,39 +1,27 @@
...
     EditText word;
-    TextView meaning;
     ImageView del;
     TextToSpeech t1;
     View v;
+    FloatingActionButton fab1, search;
     private static final String TTS_SPEAK_ID = "SPEAK";
-    String base = "https://googledictionaryapi.eu-gb.mybluemix.net";
-    final static String PARAM_QUERY = "define";
+    String press = "Press me to know the meaning";
```
Comment (atm1504) [inline app/src/main/java/com/dsciitp/shabd/Dictionary/DictionaryActivity.java:24]: What is the use of this hard coded string?

**Review by atm1504 — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/dsciitp/shabd/Dictionary/DictionaryActivity.java (hunk 2 above)
Comment (chanmol1999) [reply app/src/main/java/com/dsciitp/shabd/Dictionary/DictionaryActivity.java:24]: It's logic will be implemented later  with use of a string resource for this

**Review by atm1504 — APPROVED** [review]

File: app/src/main/java/com/dsciitp/shabd/Dictionary/DictionaryActivity.java (hunk 2 above)
Comment (atm1504) [reply app/src/main/java/com/dsciitp/shabd/Dictionary/DictionaryActivity.java:24]: okkk

**Pull request merged**